src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -131,12 +131,14 @@ public Column loadChunk(int columnX, int columnZ) {
 	@Override
 	@Nullable
 	public Column loadChunk(int columnX, int columnZ, Runnable runnable) {
-		Column column = this.getColumn(columnX, columnZ, /*Requirement.LOAD*/Requirement.LIGHT);
-		if (runnable == null) {                          // TODO: Set this to LOAD when PlayerCubeMap works
-			return column;
+		// TODO: Set this to LOAD when PlayerCubeMap works
+		if (runnable == null) {
+			return getColumn(columnX, columnZ, /*Requirement.LOAD*/Requirement.LIGHT);
```

Comment:
That comes from @xcube16's PR. getColumn takes the same Requirement for loading as does getCube, I'm not sure why we are supposed to use LIGHT there, so I didn't touch it.